charts/aad-pod-identity/README.md

```diff
@@ -162,6 +162,7 @@ The following tables list the configurable parameters of the aad-pod-identity ch
 | `mic.leaderElection.duration`            | Override leader election duration                                                                                                                                                                                | If not provided, default value is `15s`                  |
 | `mic.probePort`                          | Override http liveliness probe port                                                                                                                                                                              | If not provided, default port is `8080`                  |
 | `mic.syncRetryDuration`                  | Override interval in seconds at which sync loop should periodically check for errors and reconcile                                                                                                               | If not provided, default value is `3600s`                |
+| `mic.whiteListedIdentitiesParam`         | List of whitelisted user-defined identities that shouldn't be deleted from AAD. Note: the list should be comma separated                                                                                         | If not provided, default value is empty           |
```

Comment:
`whitelist` might be an inaccurate terminology to use, this case isn't about safe/allow vs unsafe/block identities, we basically want to exclude some identities from being deleted. I would go with something more explicit like `mic.excludeFromDeletionIdentitiesParam` or similar.